### PR TITLE
add license kiss.animate, cc #5500

### PR DESCRIPTION
**Pull request by hare1039**
license ref: https://github.com/daogurtsov/KISS.Animate/blob/master/MIT-LICENSE.txt

### CODE REVIEW

**Comment by hare1039** [comment]
@kennynaoh please check

File: ajax/libs/kiss.animate/package.json [hunk 1]
```diff
@@ -12,5 +12,6 @@
   "repository": {
     "type": "git",
     "url": "https://github.com/daogurtsov/KISS.Animate.git"
-  }
+  },
```
Comment (kennynaoh) [inline ajax/libs/kiss.animate/package.json]: @hare1039 
Please remove the blank behind the comma. Thanks!

Comment (hare1039) [inline ajax/libs/kiss.animate/package.json]: @kennynoah Should I make a new request?
Or how can I modify the content directly?

Comment (kennynaoh) [inline ajax/libs/kiss.animate/package.json]: No, you should modify it directly by using online editor upper right corner.

Comment (hare1039) [inline ajax/libs/kiss.animate/package.json]: I removed the blank. Please verify.

**Comment by PeterDaveHello** [comment]
@kennynaoh please help rebase with master branch, the code base is too old.

**Comment by kennynaoh** [comment]
No problem

**Comment by kennynaoh** [comment]
LGTM
ping @cdnjs/intern2

**Comment by pvnr0082t** [comment]
LGTM

**Pull request merged**